From 353565f59585b02ee74bb0d4e6c86099e8abcf09 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Tue, 7 Mar 2017 15:14:55 +0900
Subject: [PATCH 2408/2848] HADOOP-14087. S3A typo in pom.xml test exclusions.
 Contributed by Aaron Fabbri.

(cherry picked from commit f597f4c43e0a6e2304b9bcaf727d6d8d15a365f9)
(cherry picked from commit d2d0872f4d109b09cdc3d8e9b163544e56469621)

Change-Id: I90a2136b13c7fc2a8461f98117b617ec9f094911
---
 hadoop-tools/hadoop-aws/pom.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-aws/pom.xml b/hadoop-tools/hadoop-aws/pom.xml
index 33a2b65..8f7a431 100644
--- a/hadoop-tools/hadoop-aws/pom.xml
+++ b/hadoop-tools/hadoop-aws/pom.xml
@@ -194,7 +194,7 @@
                     <exclude>**/ITest*Root*.java</exclude>
                     <exclude>**/ITestS3AFileContextStatistics.java</exclude>
                     <exclude>**/ITestS3AEncryptionSSE*.java</exclude>
-                    <include>**/ITestS3AHuge*.java</include>
+                    <exclude>**/ITestS3AHuge*.java</exclude>
                   </excludes>
                 </configuration>
               </execution>
-- 
1.7.9.5

